socks5.py

```diff
@@ -396,6 +397,14 @@ def do_GET(s):
     return server
 
 
+# Handler for full screen button to render a full screen window.
+# Use a two-finger "slide down" gesture to close.
+def full_screen_handler(sender):
+    fs_view = ui.View()
+    fs_view.name = "Full screen"
+    fs_view.background_color = 'black'
+    fs_view.present(style='popover', hide_title_bar=True)
```

Comment:
There's also the option of adding `orientations = ['landscape']` here to force landscape mode, so that it always hides the battery indicator even while charging. Kinda hacky, but I've tested it and it does work.